src/pyfreedb/row/stmt.py

```diff
@@ -0,0 +1,280 @@
+from typing import TYPE_CHECKING, Any, Dict, Generic, List, TypeVar
+
+from pyfreedb.providers.google.sheet.base import A1CellSelector, A1Range, BatchUpdateRowsRequest
+from pyfreedb.row.base import Ordering
+from pyfreedb.row.models import Model
+
+if TYPE_CHECKING:
+    from pyfreedb.row.gsheet import GoogleSheetRowStore
+
+T = TypeVar("T", bound=Model)
+
+
+class CountStmt(Generic[T]):
+    def __init__(self, store: "GoogleSheetRowStore[T]"):
+        self._store = store
+        self._query = store._new_query_builder()
```

Comment:
I wonder why not pass in the query builder as a param rather than instantiating it here?

Reply:
Because to instantiate the query builder we need to apply the where `_rid is not null` condition directly into it, so I'm thinking to centralise it on this `_new_query_builder` method.

Reply:
I know, but my question is more about why not pass the query builder into the statement through the `__init__` parameter instead of instantiating it internally.